java-security/src/main/java/com/sap/cloud/security/token/XsuaaToken.java

```diff
@@ -159,6 +159,19 @@ public String getSubdomain() {
 		return getAttributeFromClaimAsString(EXTERNAL_ATTRIBUTE, EXTERNAL_ATTRIBUTE_ZDN);
 	}
 
+	/**
+	 * Returns subaccount identifier which is in most cases same like the
+	 * identity zone.
+	 * DO only use this for metering purposes.
+	 * DO NOT longer use this method to get the unique tenant id! For that use {@link #getZoneId()}.
+	 *
+	 * @return subaccount identifier or {@code null}
+	 */
+	@Nullable
+	public String getSubaccountId() {
+		return Optional.ofNullable(getAttributeFromClaimAsString(EXTERNAL_ATTRIBUTE, EXTERNAL_ATTRIBUTE_SUBACCOUNTID)).orElse(getZoneId());
```

Comment:
@nenaraab I used `getZoneId()` as fallback. In `XSUserInfoAdapter` `getClaimValue(ZONE_ID)` is used. Which is not the same. Can we use getZoneId() here (and in `XSUserInfoAdapter`) as fallback?

Reply:
I would do it explizit as getZoneId() is not the right semantic... but ok for now.

Reply:
No it is okay, it is just a small change. I just wanted to ask to get it consistent with `XSUserInfoAdapter`!